src/mapping/NearestNeighborBaseMapping.cpp

```diff
@@ -53,11 +51,6 @@ void NearestNeighborBaseMapping::computeMapping()
     searchSpace = input();
   }
 
-  // For log outputs
-  precice::utils::Event e2(baseEvent + ".getIndexOnVertices", precice::syncMode);
-  query::Index          indexTree(searchSpace);
-  e2.stop();
```

Comment:
Why remove the event-based stuff?

Reply:
This event is now part of the `Index` itself:
https://github.com/precice/precice/pull/1242/files#diff-4c0c54bc6aa3d8fe12a773602b1c27237290e890cd134b796ca933f85785b012R50